pyproject.toml

```diff
@@ -14,7 +14,7 @@ babel = "==2.*,>=2.0.0,>=2.5.1"
 backoff = "==1.*,>=1.0.0,>=1.4.3"
 "backports.tempfile" = ">=1.0rc1"
 beard = "==0.*,>=0.0.0,>=0.2.0"
-celery = {git = "https://github.com/michamos/celery.git", branch = "fix-redis-connection-result-consumer"}
+celery = "==4.4.2"
```

Comment:
again, I don't understand this approach. If you had specified a less strict version, poetry would have computed for you which versions of packages to get to solve the sphinx version conflict issue that you mentioned IRL. In fact, that's the whole reason to use something like poetry. If you're pinning everything, you might as well use `requirements.txt` with pinned deps.